src/Schedules.jl

```diff
@@ -114,12 +114,26 @@ struct EveryDtSchedule{T} <: AbstractSchedule
     dt::T
 
     """
-        EveryDtSchedule(dt; t_start = zero(dt))
+        EveryDtSchedule(dt; t_last = zero(dt))
 
     True every time the current time is larger than the previous time this schedule was true + dt.
     """
-    function EveryDtSchedule(dt::T; t_start::T = zero(dt)) where {T}
-        new{typeof(dt)}(Ref(t_start), dt)
+    function EveryDtSchedule(
+        dt::T;
+        t_last::T = zero(dt),
```

Comment:
Do you want to note in the doc string that t_last is always initialized to zero because the integrator time at the start_date is zero (at least for non-restarted runs)?

Reply:
Added